src/styles/index.scss

```diff
@@ -1,3 +1,35 @@
-body {
+@import 'utils/variables';
+@import 'blocks/card';
+@import 'blocks/page';
+@import 'blocks/stars';
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  font-family: $font-roboto;
```

Comment:
Ensure that the variable `$font-roboto` is defined in your SCSS files to avoid compilation errors.